src/praisonai-agents/tests/unit/test_provider_matrix.py

```
@@ -249,3 +249,33 @@ def test_a_plugin_sandbox_reaches_both_parameters(monkeypatch):
+    compute_default = None
+    try:
+        import inspect
+
+        from praisonai.integrations.compute.docker import DockerCompute
+
+        source = inspect.getsource(DockerCompute)
+        for line in source.splitlines():
+            if 'image="python:' in line:
+                compute_default = line.split('image="')[1].split('"')[0]
+                break
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)